cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -859,7 +934,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[1].editors = [StaffLockEditor];
                 } else if (xblockInfo.isSequential()) {
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
-                    tabs[1].editors = [ContentVisibilityEditor];
+                    tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
```

Comment:
@cahrens , @roderickmorales , @pomegranited - We discussed and agree with the proposal here. @roderickmorales also requests that the font size be upped to 15px if possible.